pyproject.toml

```diff
@@ -10,14 +10,14 @@ repository = "https://github.com/lmmentel/mendeleev"
 license = "MIT"
 
 [tool.poetry.dependencies]
-python = ">=3.7.1,<3.12"
+python = ">=3.8.1,<3.12"
 colorama = "^0.4.4"
 numpy = "^1.21"
 pyfiglet = "^0.8.post1"
 Pygments = "^2.8.0"
-pandas = ">=0.25.0"
+pandas = ">=1.0.0"
```

Comment:
I am a bit surprised. Is `pandas==1.0.0` really sufficient or are the recent changes of `2.0.0` required? I thought we were waiting for `2.0.0` all the time.

Reply:
Pandas 0.25 dates back to 2019 and there have been many breaking api changes along the way. Nothing directly impacting this packages afaik but I don't want to lag too much behind allowing for weird issues to come up.

Reply:
I agree but I thought you would need `pandas >=2.0.0` to be compatible with the latest `sqlalchemy` version.

Reply:
I think that sqlalchemy is now compatible with pandas 2.0.0 and older versions. I've tested both cases and it seems to work so it makes sense to keep some flexibility in versions.